### PR TITLE
Fix #126983 trendblog.euronics.de

### LINKED ISSUE #126983: trendblog.euronics.de
### Issue URL (Annoyance)
 [https://trendblog.euronics.de/tv-streaming/was-ist-mini-led-das-ist-der-unterschied-zu-oled-micro-led-und-led-91341/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ftrendblog.euronics.de%2Ftv-streaming%2Fwas-ist-mini-led-das-ist-der-unterschied-zu-oled-micro-led-und-led-91341%2F)
### Comment
Username: @Morku
> Newsletter Pop-up after 3-5 minutes

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.10.2
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Disable cache for third-party requests
DNS filtering: | server: `AdGuard DNS Non-filtering`<br>filters: `AdGuard Home with AdGuard DNS filter`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard German
Userscripts: | https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by Morku**
Fix #126983

### CODE REVIEW

File: AnnoyancesFilter/Popups/sections/subscriptions_specific.txt [hunk 1]
```diff
@@ -1060,7 +1060,7 @@ vstup.osvita.ua##.section-subscription
 news24.com##.article-subs-footer
 voelkner.de##.footer_top__content_container
 cnn.co.jp##.line-news-banner
-newdeal.by,yetkinreport.com##.mailpoet_form_popup_overlay
+trendblog.euronics.de,newdeal.by,yetkinreport.com##.mailpoet_form_popup_overlay
```
Comment (zloyden) [inline AnnoyancesFilter/Popups/sections/subscriptions_specific.txt:1063]: I see it in the code, but it does not appear on my end.
Is page scrolling blocked?

Comment (zloyden) [reply AnnoyancesFilter/Popups/sections/subscriptions_specific.txt:1063]: Nvm. I got this.

**Review by zloyden — APPROVED** [review]

**Comment by Morku** [comment]
Whoops I noticed I just blocked the Backgroundoverly, let me fix.

**Comment by Morku** [comment]
@zloyden 
Looks like, I have to Request a revert. Can you please revert to `newdeal.by,yetkinreport.com##.mailpoet_form_popup_overlay`

And add to: `parkurmerkezi.com,courseamz.com##div[class*="mailpoet_form_popup"]`
-> `trendblog.euronics.de,parkurmerkezi.com,courseamz.com##div[class*="mailpoet_form_popup"]`
Otherwise you still get the pop-up but no grey overlay.

<details><summary>Screenshot-1:</summary>

trend


</details><br/>

**Comment by zloyden** [comment]
Of course, it's done:)

**Pull request merged**